cmd/tapcli/assets.go

```diff
@@ -142,6 +145,30 @@ func parseAssetType(ctx *cli.Context) (taprpc.AssetType, error) {
 	}
 }
 
+func parseMetaType(metaType string,
+	metaBytes []byte) (taprpc.AssetMetaType, error) {
+
+	switch metaType {
+
+	case "opaque":
```

Comment:
Doesn't this change disallow people to play around with custom meta types on the CLI?
Maybe in the default case we should try to parse the type as a number and then use that and only fail if it's not a valid number? So people can still experiment with types > 1.

Reply:
Ah sure, modified to retain that ability, but still allow these first class aliases.